spinalcordtoolbox/scripts/sct_apply_transfo.py

```diff
@@ -290,13 +291,18 @@ def apply(self):
 
         # Copy affine matrix from destination space to make sure qform/sform are the same
         printv("Copy affine matrix from destination space to make sure qform/sform are the same.", verbose)
-        im_src_reg = Image(fname_out)
-        im_src_reg.copy_qform_from_ref(Image(fname_dest))
-        im_src_reg.save(verbose=0)  # set verbose=0 to avoid warning message about rewriting file
+        # mmap=False prevents bus error, see https://github.com/nipreps/niworkflows/issues/246
+        im_src_reg = nib.load(fname_out, mmap=False)
+        im_dest = nib.load(fname_dest)
+        qform, qform_code = im_dest.header.get_qform(coded=True)
+        sform, sform_code = im_dest.header.get_sform(coded=True)
+        im_src_reg.set_qform(qform, int(qform_code))
+        im_src_reg.set_sform(sform, int(sform_code))
+        nib.save(im_src_reg, fname_out)
```

Comment:
As an alternative to using `nib.load` directly like this, we could instead modify the internals of SCT's `Image` class to behave more like `Nifti1Image` (e.g. not load the data immediately).

But, this would have a far wider effect on SCT, something that is both beneficial and risky. Still mulling this over...